subsys/bluetooth/shell/bt.c

```diff
@@ -694,8 +694,6 @@ static void connected(struct bt_conn *conn, uint8_t err)
 		}
 	}
 
-	default_conn = bt_conn_ref(conn);
```

Comment:
I can see the bug with the ref getting taken twice for `ROLE_PERIPHERAL`, but where is `default_conn` getting set for `ROLE_CENTRAL`?

Reply:
> I can see the bug with the ref getting taken twice for `ROLE_PERIPHERAL`, but where is `default_conn` getting set for `ROLE_CENTRAL`?

Nevermind, I'm just blind.